swaglyrics/__main__.py

```diff
@@ -42,8 +47,15 @@ def main():
 		app.run(port=port)
 
 	elif args.cli:
-		song = spotify.song()  # get currently playing song
-		artist = spotify.artist()  # get currently playing artist
+		if args.song is None and args.artist is None:
+			song = spotify.song()  # get currently playing song
+			artist = spotify.artist()  # get currently playing artist
+		else:
+			song = args.song  # get song from command line argument
```

Comment:
these are over-indented

Reply:
why is it giving that over indented error at those lines

Isn't it indented properly as per the other lines?
![image](https://user-images.githubusercontent.com/32815509/55646299-6e90cc80-57f8-11e9-8da3-93578692d530.png)